docs/imagecustomizer/cli.md

```diff
@@ -31,7 +31,7 @@ The file path to write the final customized image to.
 
 The image format of the the final customized image.
 
-Options: vhd, vhd-fixed, vhdx, qcow2, raw, and iso.
+Options: vhd, vhd-fixed, vhdx, qcow2, raw, iso, and cosi.
 
 At least one of `--output-image-format` and `--output-split-partitions-format` is
```

Comment:
Do we plan to deprecate `output-split-partitions-format` in this PR or a separate one?

Reply:
I think we should have 1-2 releases where we support both APIs. That will make it easier for customers to switch at their own pace.